pkg/pillar/hypervisor/xen.go

```diff
@@ -144,6 +144,9 @@ func (ctx xenContext) Setup(status types.DomainStatus, config types.DomainConfig
 
 	// finally we can start it up
 	spec.Get().Process.Args = []string{"/etc/xen/scripts/xen-start", status.DomainName, file.Name()}
+	if config.MetaDataType == types.MetaDataOpenStack {
+		spec.Get().Process.Args = append(spec.Get().Process.Args, "overwrite")
```

Comment:
The name overwrite here doesn't make much sense -- why can't we call it something like `smbios_product`?